### DISCUSSION TITLE
How to unload ip adapter?

### DISCUSSION THREAD

**Discussion by wilfrediscoming** [huggingface/diffusers#7040] [Q&A] [answered]
How to unload ip adapter?

`pipeline.load_ip_adapter("h94/IP-Adapter", subfolder="models", weight_name="ip-adapter_sd15.bin")`

I used .load_ip_adapter, but what if laterwards I want to unload the ipadapter?

**Comment by asomoza** [comment, marked as answer]
Like this:

```python
pipeline.unload_ip_adapter()
```
